src/Schema/Events/Click.ts

```diff
@@ -1742,4 +1742,61 @@ export interface ClickedVerifiedRepresentative {
   context_page_owner_id: string
   destination_page_owner_type: PageOwnerType
   destination_page_owner_id: string
+}
+
+/**
+ * A user clicks one of the related categories(genes) in the artist about tab
+ *
+ *  This schema describes events sent to Segment from [[clickedRelatedCategory]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedRelatedCategory",
+ *    context_module : "Young British Artists",
+ *    context_screen_owner_type: "Artwork",
+ *    context_screen_owner_id: "58de681f275b2464fcdde097",
+ *    context_screen_owner_slug: "damien-hirst",
+ *    destination_screen_owner_type: "Gene"
+ *    destination_screen_owner_id: "58de681f275b2464fcdde097"
+ *    subject: ""
+ *  }
+ * ```
+ */
+export interface ClickedRelatedCategory {
+  action: ActionType.clickedRelatedCategory
+  context_module: ContextModule
+  context_screen_owner_type: PageOwnerType
```

Comment:
Aren't these supposed to be `page` instead of `screen` fields for the web and `screen` when comes to mobile?

Reply:
Tap events are fine as far as I checked, they use `screen` fields

Reply:
you're right, it's a copy paste from app that I forgot to amend